presto-product-tests/src/main/java/com/facebook/presto/tests/CommentTableTests.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.teradata.tempto.AfterTestWithContext;
+import com.teradata.tempto.BeforeTestWithContext;
+import com.teradata.tempto.ProductTest;
+import com.teradata.tempto.Requires;
+import com.teradata.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableNationTable;
+import com.teradata.tempto.query.QueryResult;
+import io.airlift.log.Logger;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.TestGroups.ALTER_TABLE;
+import static com.facebook.presto.tests.TestGroups.SMOKE;
+import static com.teradata.tempto.query.QueryExecutor.query;
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+
+@Requires(ImmutableNationTable.class)
+public class CommentTableTests
+        extends ProductTest
+{
+    private static final String TABLE_NAME = "comment_test";
+
+    @BeforeTestWithContext
+    @AfterTestWithContext
+    public void dropTestTables()
+    {
+        try {
+            query(format("DROP TABLE IF EXISTS %s", TABLE_NAME));
+        }
+        catch (Exception e) {
+            Logger.get(getClass()).warn(e, "failed to drop table");
+        }
+    }
+
+    @Test(groups = {ALTER_TABLE, SMOKE})
+    public void renameTable()
+    {
+        String createTableSql = format("" +
+                        "CREATE TABLE hive.default.%s (\n" +
+                        "   c1 bigint\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'RCBINARY'\n" +
+                        ")",
+                TABLE_NAME);
+
+        query(createTableSql);
```

Comment:
what about test, when table is created with some comment and then you update the comment